lib/offsite_payments/integrations/paytm.rb

```diff
@@ -207,6 +207,8 @@ def checksum_ok?
             @message = 'Return checksum not matching the data provided'
             false
           end
+        rescue
```

Comment:
Blanket rescue seems heavy handed for the given example, why not an early `return false if checksum.nil?` instead? This way we keep visibility in case a legit error surfaces in this block.

Reply:
They seem to have other fields missing but I don't mind the early return for now!